pages/docs/reference/properties.md

```diff
@@ -115,6 +115,29 @@ var setterWithAnnotation: Any? = null
 ```
 </div>
 
+
+### Computed Properties
+
+When using a computed property, the value is computed once and stored. This behavior changes when using adding `get()`. The `get()` causes the value to be computed on every access.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+``` kotlin
+val isEmpty get() = this.size == 0 // Computed on every access.
+val isEmpty = this.size == 0 // Computed once and stored.
+```
+</div>
+
+When using a `var` the behavior is almost the same except that the computed value is stored until it changes or is overwritten.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+``` kotlin
+var isEmpty = false
```

Comment:
could you please avoid using backing fields here, which are discussed below. It looks like you may have a setter declared too